pom.xml

```diff
@@ -30,6 +30,29 @@
             <name>Richard Körber</name>
         </developer>
     </developers>
+    <distributionManagement>
+        <!--<snapshotRepository>-->
+            <!--<id>sun-artifactory-snapshots</id>-->
+            <!--<name>sun-snapshots</name>-->
+            <!--<url>https://repo.artifacts.weather.com/sun-snapshot-local</url>-->
+        <!--</snapshotRepository>-->
+        <snapshotRepository>
+            <id>ssg-dev-repository-snapshot</id>
+            <name>SSG Snapshots</name>
+            <url>scp://dev-user@jenkinsutil.dev.sun.weather.com:/home/dev-user/pub/mvn/ssg-snapshots</url>
+            <layout>default</layout>
+        </snapshotRepository>
+        <!--<repository>-->
+            <!--<id>sun-nexus-release</id>-->
+            <!--<name>sun-nexus-release</name>-->
+            <!--<url>https://nexus.dev.sun.weather.com/nexus/content/repositories/sun-releases</url>-->
+        <!--</repository>-->
+        <!--<snapshotRepository>-->
+            <!--<id>sun-nexus-snapshots</id>-->
+            <!--<name>sun-nexus-snapshots</name>-->
+            <!--<url>https://nexus.dev.sun.weather.com/nexus/content/repositories/sun-snapshots</url>-->
+        <!--</snapshotRepository>-->
+    </distributionManagement>
```

Comment:
I was planning on creating another branch without it and submit the PR agains origin from there